src/server/db/schema/officers.ts

```diff
@@ -0,0 +1,24 @@
+import { relations } from 'drizzle-orm';
+import { pgTable, text, boolean, primaryKey } from 'drizzle-orm/pg-core';
+import { club } from './club';
+
+export const officers = pgTable(
+  'officers',
+  {
+    id: text('id').notNull(),
+    clubId: text('club_id')
+      .notNull()
+      .references(() => club.id),
+    name: text('name').notNull(),
+    position: text('position').notNull(),
+    image: text('image').default('/nebula-logo.png').notNull(),
+    isPresident: boolean('is_president').default(false).notNull(),
+  },
+  (table) => ({
+    pk: primaryKey(table.clubId, table.id),
```

Comment:
Why are we doing composite key here?

Reply:
Now that I'm looking at it, there really isn't a reason for a composite key. I'll get rid of that.

Reply:
Sounds good. I was thinking that maybe it was supposed to be linked with the `userId` but it's not a column in the table so prob just leave it out